test/integration/targets/scaleway_lb_facts/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+- name: Create load-balancer
+  scaleway_lb:
+    state: present
+    name: '{{ name }}'
+    region: '{{ scaleway_region }}'
+    organization_id: '{{ scw_org }}'
+    description: '{{ description }}'
+    tags: '{{ tags }}'
+    wait: true
+
+- name: Get lb informations and register it in a variable
+  scaleway_lb_facts:
+    region: fr-par
+  register: lb_fr_par
+
+- name: Display lb_fr_par1 variable
+  debug:
+    var: lb_fr_par
+
+- name: Ensure retrieval of ips facts is success
+  assert:
+    that:
+      - lb_fr_par is success
+
+- name: Ensure that the name is the one we configured.
+  assert:
+    that:
+      - '{{ lb_fr_par.ansible_facts.scaleway_lb_facts[0].name == name }}'
```

Comment:
`that` parameter doesn't need brackets.

Reply:
It fails on my machine if I remove them.

Reply:
Fixed